steps/bitrise-step-qyrus-mobility-test-runner-android/0.0.1/step.yml

```diff
@@ -0,0 +1,126 @@
+title: qyrus-mobility-test-runner-android
+summary: Upload apk and ipa to Qyrus platform, triggers the tests.
+description: |
+  This step uploads the apk or ipa to Qyrus test platform, runs the test on real devices.
+website: https://github.com/qyrusai/bitrise-step-qyrus-mobility
+source_code_url: https://github.com/qyrusai/bitrise-step-qyrus-mobility
+support_url: https://github.com/qyrusai/bitrise-step-qyrus-mobility/issues
+published_at: 2021-11-19T19:16:12.819674+05:30
+source:
+  git: https://github.com/QyrusAI/bitrise-step-qyrus-mobility-test-runner-android.git
+  commit: 82cacba24f15f7c38a2e86cc66230840506c7679
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
```

Comment:
You are not using those listed dependencies but npm inside the script.

Reply:
thought it might require git to pull the code on the bitrise agents. the comment makes sense. I will remove unused dependencies.